test/visual/stories.js

```diff
@@ -136,6 +136,27 @@ module.exports = [
     'checkbox--emphasized-disabled-checked',
     'checkbox--tab-index-example',
     'checkbox--vertical-tab-index-example',
+    'circle-loader--default',
+    'circle-loader--over-background',
```

Comment:
Looks like these two out-of-date stories snuck in somewhere... Available below as `progress-circle--*`.